lib/debug/deprecations.js

```diff
@@ -10,6 +10,19 @@ function deprecation(options) {
   };
 }
 
-const DEPRECATIONS = {};
+const DEPRECATIONS = {
+  V1_ADDON_CONTENT_FOR_TYPES: deprecation({
+    for: 'ember-cli',
+    id: 'ember-cli.v1-addon-content-for-types',
+    since: {
+      available: '6.3.0',
+    },
+    until: '7.0.0',
+    url: 'https://deprecations.emberjs.com/id/v1-addon-content-for-types',
```

Comment:
@bertdeblock 

Page not found, 404 error:

<img width="994" alt="image" src="https://github.com/user-attachments/assets/0c7be24f-6948-4d4b-bdd5-9d350365f222" />

Reply:
Still TODO indeed.

For now, see https://github.com/emberjs/rfcs/blob/advance-rfc-1029/text/1029-deprecate-app-prefix.md#deprecation-guide-content